### PR TITLE
Create an opprtunity addition form with edit the data base and succes…

### PR DESCRIPTION
**Pull request by youssef910**
…fuly add data to data base   ,added cities table and skills table and edited opportunity table ,add api connected to database

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -27,12 +28,20 @@ const Routes = () => {
...
 
         <div>
-          <Route path="/" exact component={Home} />
-          <Route path="/about/" component={About} />
-          <Route path="/status/" component={Status} />
+          <Switch>
+            <Route path="/" exact component={Home} />
+            <Route path="/about/" component={About} />
+            <Route path="/status/" component={Status} />
+            <Route Path="/createOpprtunity" component={CreateOppForm} />
```
Comment (kabaros) [inline client/src/index.js]: misspelled - also route should `create-opportunity` (make sure the link is updated too)

File: server/api/cities.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const router = express.Router();
+const cities = require("../services/database/cities");
+
+/**
+ * The route here will be: /cities/ (remember the prefix users is defined in api/index.js)
```
Comment (kabaros) [inline server/api/cities.js]: nice one!

File: server/api/index.js [hunk 3]
```diff
@@ -3,8 +3,14 @@ const router = express.Router();
...
 const status = require("./status");
+const cities = require("./cities");
+const skills = require("./skills");
+const newOpportunity = require("./newOpportunity");
 
 router.use("/users", users);
 router.use("/status", status);
+router.use("/cities", cities);
+router.use("/skills", skills);
+router.use("/newOpportunity", newOpportunity);
```
Comment (kabaros) [inline server/api/index.js]: rename them to `/opportunities` and make sure the client is also updated to `POST /opportunities` .. also update the file name to be `opportunities.js`

File: server/api/skills.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const router = express.Router();
+const skills = require("../services/database/skills");
+
+/**
+ * The route here will be: /cities/ (remember the prefix users is defined in api/index.js)
```
Comment (kabaros) [inline server/api/skills.js]: it is not cities .. it is skills

File: server/db/populate-db.sql [hunk 5]
```diff
@@ -1,30 +1,105 @@
 -- Users seed data
 -- Note: Do NOT add passwords you use in real life. Passwords are now saved as clear text. This is very bad from security point of view. We will have a task to encrypt the passwords
+INSERT INTO cities
```
Comment (kabaros) [inline server/db/populate-db.sql]: weird formatting - use `pgformatter` extension for formatting the file

**Comment by youssef910** [comment]
responded to all comments

File: server/db/recreate-schema.sql [hunk 6]
```diff
@@ -40,10 +42,20 @@ CREATE TYPE status_type AS ENUM (
   'approved'
 );
 
+CREATE TABLE skills (
```
Comment (kabaros) [inline server/db/recreate-schema.sql]: you need to also drop this if it exits

File: client/package.json [hunk 7]
```diff
@@ -7,6 +7,7 @@
     "react-dom": "^16.8.6",
     "react-router-dom": "^5.0.1",
     "react-scripts": "3.0.1",
+    "semantic-ui-calendar-react": "^0.15.3",
```
Comment (kabaros) [inline client/package.json]: why did you add `semantic-ui-calendar-react`

File: server/db/recreate-schema.sql (hunk 6 above)
Comment (youssef910) [reply server/db/recreate-schema.sql]: it is droped at line 12
